cql3/query_processor.cc

```diff
@@ -668,10 +668,14 @@ struct internal_query_state {
     bool more_results = true;
```

Comment:
All those 4 listed changes should be in separate commit each.

Reply:
I don't think there's any value in doing that (I'm for splitting into small commits in general, but in that particular case it would do more harm than good IMO)

Reply:
It would be easier to review. Bundling together unrelated changes makes more harm.

Reply:
Isn't this change sufficiently small and "obvious"?